### ISSUE TITLE
Improve name for "sex" argument in wrightfisher model

### ISSUE THREAD

**Issue by eaton-lab** [elissasoroj/shadie#21]
This example would make it more clear that the type is binary and it is defining separate versus non-separate sexes. It is also more clear that this does not mean sex versus asexual, or other such things.
 
```python
model.reproduction.wright_fisher(separate_sexes=True)
```
Instead of 
```python
model.reproduction.wright_fisher(sex=True)
```

**Comment by eaton-lab** [comment]
I see that the 'sex' argument exists for the `wright_fisher_haploid_clonal` model. Since it only reproduces clonally this argument should probably not exist.

**Comment by elissasoroj** [comment]
Fixed in [v1.0.1 release](https://github.com/elissasoroj/shadie/releases/tag/v1.0.1)
